src/redux/store.ts

```diff
@@ -0,0 +1,14 @@
+import { configureStore } from "@reduxjs/toolkit";
+import chefsReducer from "./slices/chefsSlice";
+import restaurantsReducer from "./slices/restaurantsSlice";
+import dishesReducer from "./slices/dishesSlice";
+
+const store = configureStore({
+  reducer: {
+    chefs: chefsReducer,
+    restaurants: restaurantsReducer,
+    dishes: dishesReducer,
+  },
```

Comment:
same as route, we dont need a 3 slices for same action. 
if we add a 2000 collection we will create for each 1 slice?